tasks/redhat.yml

```
@@ -0,0 +1,20 @@
+- name: Add firewalld rules
```

Can this please done as a loop instead, e.g.

```suggestion
- name: Add firewalld rules
  firewalld:
    port: "{{ item }}"
    permanent: yes
    state: enabled
  loop:
    - 2181/tcp
    - 2888/tcp
    - 3888/tcp
  notify: Reload firewalld
```